WearOAuth/gradle/libs.versions.toml

```diff
@@ -1,18 +1,21 @@
 [versions]
 android-gradle-plugin = "8.4.2"
+androidx-compose-bom = "2024.05.00"
 androidx-activity = "1.9.0"
 androidx-lifecycle = "2.8.1"
 androidx-test = "1.5.2"
 androidx-wear-compose = "1.3.1"
-androidx-wear-tiles = "1.3.0"
-androidx-wear-watchface = "1.2.1"
 compose = "1.6.7"
 compose-compiler = "1.5.14"
 org-jetbrains-kotlin = "1.9.24"
 org-jetbrains-kotlinx = "1.8.1"
 ktlint = "0.50.0"
 compose-material3 = "1.0.0-alpha23"
-horologist-compose = "0.6.12"
+horologist = "0.6.12"
```

Comment:
Since we are on 0.6.12, then we are already on compose 1.7-betas and wear compose 1.4 betas

Reply:
done